### PR TITLE
Upgrade Flickr4Java version (and fix incompatabilities)

### PR DESCRIPTION
**Pull request by seehamrun**

### CODE REVIEW

**Comment by wmorland** [comment]
Nice, looks good to me but happy to defer to @philipsharp

File: extensions/data-transfer/portability-data-transfer-flickr/build.gradle [hunk 1]
```diff
@@ -23,6 +23,7 @@ dependencies {
     compile project(':portability-spi-transfer')
 
     compile("com.flickr4java:flickr4java:${flickrVersion}")
+    compile("org.scribe:scribe:1.3.7")
```
Comment (philipsharp) [inline extensions/data-transfer/portability-data-transfer-flickr/build.gradle:26]: The new version of flickr4java requires Scribe 6.9.0. Why is this dependency being added here?

File: extensions/data-transfer/portability-data-transfer-flickr/src/main/java/org/datatransferproject/datatransfer/flickr/photos/FlickrUtils.java [hunk 2]
```diff
@@ -21,6 +21,7 @@
 import com.flickr4java.flickr.Flickr;
 import com.flickr4java.flickr.FlickrException;
 import com.flickr4java.flickr.auth.Auth;
+import com.github.scribejava.core.model.OAuth1Token;
```
Comment (philipsharp) [inline extensions/data-transfer/portability-data-transfer-flickr/src/main/java/org/datatransferproject/datatransfer/flickr/photos/FlickrUtils.java:24]: Is this import needed if the class isn't being references?

File: extensions/data-transfer/portability-data-transfer-flickr/src/main/java/org/datatransferproject/datatransfer/flickr/photos/FlickrUtils.java [hunk 3]
```diff
@@ -21,6 +21,7 @@
...
 import org.datatransferproject.types.transfer.auth.AuthData;
 import org.datatransferproject.types.transfer.auth.TokenSecretAuthData;
 import org.scribe.model.Token;
```
Comment (philipsharp) [inline extensions/data-transfer/portability-data-transfer-flickr/src/main/java/org/datatransferproject/datatransfer/flickr/photos/FlickrUtils.java:27]: Also this import doesn't look to be needed anymore.

**Pull request closed without merging**